javascript-conditionals/main.js

```diff
@@ -0,0 +1,118 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+function isUnderFive(number) {
+  return number < 5;
+}
+console.log('4 is under five:', isUnderFive(4));
+console.log('10 is under five:', isUnderFive(10));
+console.log('5 is under five:', isUnderFive(5));
+
+function isEven(number) {
+  return number % 2 === 0;
+}
+console.log('4 is even:', isEven(4));
+console.log('10 is even:', isEven(10));
+console.log('5 is even:', isEven(5));
+
+function startsWithJ(string) {
+  return string.charAt(0) === 'J';
+}
+console.log('Javascript starts with J:', startsWithJ('Javascript'));
+console.log('PHP starts with J:', startsWithJ('PHP'));
+console.log('HTML starts with J:', startsWithJ('HTML'));
+console.log('Java starts with J:', startsWithJ('Java'));
+console.log('Kotlin starts with J:', startsWithJ('Kotlin'));
+console.log('C# starts with J:', startsWithJ('C#'));
+
+function isOldEnoughToDrink(person) {
+  return person.age >= 21;
+}
+const bart = {
+  name: 'Bartholomew JoJo Simpson',
+  age: 10
+};
+console.log(bart.name + ' ' + 'is old enough to drink:', isOldEnoughToDrink(bart));
+
+function isOldEnoughToDrive(person) {
+  return person.age >= 16;
+}
+const homer = {
+  name: 'Homer Jay Simpson',
+  age: 39
+};
+console.log(homer.name + ' ' + 'is old enough to drive:', isOldEnoughToDrive(homer));
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return (person.age >= 21) && !(person.age >= 16);
+  // Should return false no matter what age is put in.
+}
+const dan = {
+  name: 'Daniel Hahn',
+  age: 25
+};
+console.log(dan.name + ' ' + 'is old enough to drink and drive:', isOldEnoughToDrinkAndDrive(dan));
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if ((pH > 0) && (pH < 7)) {
```

Comment:
What happens if `pH` equals `0` or `14`?

Reply:
Oh brain fart - I added the equal to on the wrong else if statement. Correcting it again!